Dockerfile

```diff
@@ -4,16 +4,15 @@ WORKDIR /github.com/layer5io/meshery-istio
 ADD . .
 RUN GOPROXY=direct GOSUMDB=off go build -ldflags="-w -s" -a -o /meshery-istio .
 RUN find . -name "*.go" -type f -delete; mv istio /
-RUN wget -O /istio.tar.gz https://github.com/istio/istio/releases/download/1.7.3/istio-1.7.3-linux-amd64.tar.gz
+RUN wget -O /istio.tar.gz https://github.com/istio/istio/releases/download/1.5.1/istio-1.5.1-linux.tar.gz
```

Comment:
@kumarabd are you intentionally bundling this older version?

Reply:
Yes lee, this will have no effect on the latest one, but will enable us to install older versions